### ISSUE TITLE
Attempting to connect to ODBC for MS SQL

### ISSUE THREAD

**Issue by Morvalyn** [JuliaDB/ODBC.jl#75]
Failure attempting to MS SQL

julia> ODBC.connect("NolanTest")
ODBC Connection Object
----------------------
Connection Data Source: NolanTest
NolanTest Connection Number: 1
Contains resultset(s)? No

julia> query("select * from StoreProducts")
ERROR: invalid UTF-8 sequence
 in convert at utf8.jl:162 (repeats 2 times)

or

julia> query("select * from StoreProducts")
[ODBC] 42000: [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Incorrect sy
ntax near '@'.
ERROR: [ODBC]: SQLExecDirect failed; Return Code: SQL_DEFAULT_ERROR
 in error at error.jl:21 (repeats 2 times)

**Comment by Ken-B** [comment]
I get the same issues with Julia v0.3.6 on Windows 7. This breaks the package for me.

I use a cumbersome workaround calling pyodbc for those in need of an immediate solution:

```jl
using PyCall
@pyimport pyodbc
cnxn  = pyodbc.connect("DSN=my_dsn_name")
cursor = cnxn[:cursor]()
cex = cursor[:execute]("SELECT * FROM LAI_App.dbo.results")
res = cex[:fetchall]()
```

And I do a final `[collect(res[i]) for i=1:length(res)]` to convert the data to Julia types.

For me a connection to Microsoft SQL is an essential of Julia. There seems to be some related issues and PR's on this, so I understand it's not easy to fix.

**Comment by quinnj** [comment]
should be resolved by the overhaul 7ae7ebfac60ed3ffb1ccefd552993a3073f8b77b
